src/playlist-loader.js

```diff
@@ -632,11 +715,9 @@ export default class PlaylistLoader extends EventTarget {
 
       this.src = resolveManifestRedirect(this.handleManifestRedirects, this.src, req);
 
-      const manifest = parseManifest({
+      const manifest = this.parseManifest_({
```

Comment:
move parseManifest to `this.parseManifest` as we weren't previously passing all the options in both calls.